tests/async.test.tsx

```diff
@@ -27,7 +27,7 @@ it('does not show async stale result', async () => {
     const [count, setCount] = useAtom(countAtom)
     const onClick = async () => {
       setCount((c) => c + 1)
-      await new Promise((r) => setTimeout(r, 10))
+      await new Promise((r) => setTimeout(r, 1))
```

Comment:
@Thisen Unfortunately, this is causing the CI failure several times, so let me revert it.